syntax = "proto3";

package fraud_detection;

service FraudDetectionService {
    rpc CheckFraud (FraudRequest) returns (FraudResponse);
}

message FraudRequest {
    string card_number = 1;
    float order_amount = 2;
}

message FraudResponse {
    bool is_fraud = 1;
}



service HelloService {
    rpc SayHello (HelloRequest) returns (HelloResponse);
}

message HelloRequest {
    string name = 1;
}

message HelloResponse {
    string greeting = 1;
}